indico/modules/events/abstracts/models/notification_templates.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+from indico.util.notification_templates import NotificationTemplateBase
+
+
+def _get_next_position(context):
+    """Get the next abstracts template position for the event."""
+    event_id = context.current_parameters['event_id']
+    res = db.session.query(db.func.max(AbstractsNotificationTemplate.position)).filter_by(event_id=event_id).one()
+    return (res[0] or 0) + 1
+
+
+class AbstractsNotificationTemplate(NotificationTemplateBase, db.Model):
+    """Represents an email notification template for abstracts"""
+
+    __tablename__ = 'notification_templates'
+    __auto_table_args = ({'schema': 'event_abstracts'})
```

Comment:
use regular `__table_args__` if you don't need to get some from a mixin/base. right now they don't do anything because you don't have the regular one with the `auto_table_args` call.